pkgs/os-specific/linux/libnvme/default.nix

```diff
@@ -35,6 +35,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/linux-nvme/libnvme/commit/ff742e792725c316ba6de0800188bf36751bd1d1.patch";
       hash = "sha256-IUjPUBmGQC4oAKFFlBrjonqD2YdyNPC9siK4t/t2slE=";
     })
+    # included in next release
+    (fetchpatch {
+      url = "https://github.com/linux-nvme/libnvme/pull/727.patch";
+      hash = "sha256:0SrR1++QTXNq+X4YOpW3JzEjFdSSBdqPl/xI2zdcvTs=";
```

Comment:
is it normal that this hash looks a bit different than the others (`:` instead of `-`)? I assume it doesn't matter much, idk.